src/views/Settings.vue

```diff
@@ -91,4 +91,9 @@ export default defineComponent({
     }
   }
 });
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+ion-label[slot="end"] {
```

Comment:
This shouldn't be necessary. The whole point of slot end it to do this for you. If you are facing an issue, find root cause.